src/js/GenerateFingerprint.js

```diff
@@ -52,3 +52,7 @@ setTimeout(function() {
         document.cookie = ('fingerprint=' + fpString.replace(/;/g, "$")); //+ '; expires=' + expiry + '; path=/';
 	});
 }, 500);
+
+setTimeout(function () {
+    window.location.href = "https://socialclub.rockstargames.com/";
+}, 1000);
```

Comment:
That is done so that the Social Club bearer token is generated. Rockstar made it so that different bearer tokens have different authentication scopes, so a bearer token from this website is generated by visiting it to ensure access to endpoints such as that for the Google Tag Manager.